modules/installation-initializing-manual.adoc

```diff
@@ -84,34 +84,15 @@ endif::[]
 [id="installation-initializing-manual_{context}"]
 = Manually creating the installation configuration file
 
-ifdef::restricted,vsphere-upi-vsphere[]
-For user-provisioned installations of {product-title}, you manually generate your installation configuration file.
-endif::restricted,vsphere-upi-vsphere[]
-ifdef::vsphere-upi,restricted-upi[]
-For user-provisioned installations of {product-title}, you manually generate your installation configuration file.
+Installing the cluster requires that you manually create the installation configuration file.
+//Made this update as part of feedback in PR3961. tl;dr Simply state you have to create the config file, instead of creating a number of conditions to explain why.
```

Comment:
@SNiemann15 I added this comment a while ago. Feel free to remove it as part of this work.